.github/workflows/units.yml

```diff
@@ -0,0 +1,68 @@
+---
+name: unit tests
+
+on: [workflow_call]  # allow this workflow to be called from other workflows
+
+jobs:
+  unit-source:
+    uses: ansible-network/github_actions/.github/workflows/unit_source.yml@main
+    with:
+      matrix_exclude: >-
```

Comment:
Do we actually need this anymore? It seems like we should be able to just use the default excludes.

Reply:
This change won't work because changes in this repo aren't merged with the defaults, they completely overwrite. What I'm suggesting is that we just don't provide a `matrix_exclude` at all for the unit test workflow and just use whatever that action defaults to. I don't think we can do the same yet for sanity because of ansible 2.9.